frontend/public/locales/en/common.json

```diff
@@ -192,5 +191,33 @@
     "volumeMounts": "Volume Mounts",
     "warning": "warning",
     "withError": "with error",
-    "yaml": "YAML"
-}
+    "key27": "  Getting Started",
```

Comment:
1. Please confirm if there is any other way than a line by line translation.
2. May want to investigate if we should even have that page
3. We may want to put these in a separate json file, or at least not under the common one.

Reply:
There seems to be a 'one line' way of having json 
This is just a quick test, but using `\n`
```
"key2": " **Demos234** say: {{val}} <br/> \n ## Build your own pipeline with \n * TensorFlow Extended (TFX) [SDK](https://www.tensorflow.org/tfx/guide) with end-to-end ML Pipeline Template ([Open TF 2.1 Notebook](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb)) \n      * Kubeflow Pipelines [SDK](https://www.kubeflow.org/docs/pipelines/sdk/) \n    <br/>"
```

Reply:
I will create a gettingstarted.json to this page and try to load it in one paragraph.